cmd/cli/cmd/migrate.go

```diff
@@ -96,28 +116,47 @@ func NewMigrate() *cobra.Command {
 			okCheck := color.New(color.FgGreen).FprintlnFunc()
 			okCheck(cmd.OutOrStdout(), "\nMigration Succeeded 🎉")
 
-			if opts.SkipConnect {
+			instanceURL := fmt.Sprintf("%s/instances/%s", opts.CloudDashboardURL, instanceID)
+
+			if opts.SkipOpenBrowser {
+				fmt.Println(heredoc.Docf(`
+				 Visit the URL to see your instance details:
+				 %s
+		`, instanceURL))
 				return nil
 			}
-			return browser.OpenURL(fmt.Sprintf("%s/instances/%s", opts.CloudDashboardURL, instanceID))
+
+			fmt.Println(heredoc.Docf(`
+			If your browser didn't open automatically, visit the URL to see your instance details:
+				 %s
+		`, instanceURL))
+			return browser.OpenURL(instanceURL)
 		},
 	}
 
 	flags := login.Flags()
 	flags.StringVar(&opts.Token, "token", "", "Botkube Cloud authentication token")
 	flags.StringVar(&opts.InstanceName, "instance-name", "", "Botkube Cloud Instance name that will be created")
-	flags.StringVar(&opts.CloudAPIURL, "cloud-api-url", "https://api.botkube.io", "Botkube Cloud API URL")
+	flags.StringVar(&opts.CloudAPIURL, "cloud-api-url", "https://api.botkube.io/graphql", "Botkube Cloud API URL")
 	flags.StringVar(&opts.CloudDashboardURL, "cloud-dashboard-url", "https://app.botkube.io", "Botkube Cloud URL")
 	flags.StringVarP(&opts.Label, "label", "l", "app=botkube", "Label of Botkube pod")
 	flags.StringVarP(&opts.Namespace, "namespace", "n", "botkube", "Namespace of Botkube pod")
 	flags.BoolVarP(&opts.SkipConnect, "skip-connect", "q", false, "Skips connecting to Botkube Cloud after migration")
+	flags.BoolVar(&opts.SkipOpenBrowser, "skip-open-browser", false, "Skips opening web browser after migration")
+	flags.BoolVar(&opts.AutoUpgrade, "auto-upgrade", false, "Automatically upgrades Botkube instance without additional prompt")
+	flags.StringVar(&opts.ConfigExporter.Registry, "cfg-exporter-image-registry", "ghcr.io", "Config Exporter job image registry")
+	flags.StringVar(&opts.ConfigExporter.Repository, "cfg-exporter-image-repo", "kubeshop/botkube-config-exporter", "Config Exporter job image repository")
+	flags.StringVar(&opts.ConfigExporter.Tag, "cfg-exporter-image-tag", DefaultImageTag, "Config Exporter job image tag")
+	flags.DurationVar(&opts.ConfigExporter.PollPeriod, "cfg-exporter-poll-period", 1*time.Second, "Config Exporter job poll period")
+	flags.DurationVar(&opts.ConfigExporter.Timeout, "cfg-exporter-timeout", 1*time.Minute, "Config Exporter job timeout")
+	flags.BoolVarP(&opts.Debug, "debug", "d", false, "Turn on debug logging")
```

Comment:
On my PR I introduce a global `verbose` flag, so it will be easy to check across the whole code-base whether we should print more info. It should also make it consistent in case we reuse some components between commands, as the one global value will be used.

Additionally, I introduced an option to specify verbose level, the same what we had in the previous project. It can be useful when we run some GraphQL calls, and with `trace` we can print them too, but not always when `-v` is specified as it could be too verbose.